Tests/test_PDB_PDBIO.py

```diff
@@ -26,6 +26,7 @@
 from Bio.PDB import Select
 from Bio.PDB.PDBExceptions import PDBConstructionWarning
 from Bio.PDB.PDBExceptions import PDBIOException
+from Bio.PDB.PDBIO import _MAX_B_FACTOR, PDBIOWarning
```

Comment:
I'd import the warning on it's own, like above with `PDBIOException` (and from the Exceptions module)